cdap-common/bin/cdap

```diff
@@ -86,7 +86,7 @@ case ${1} in
     echo
     fi
     echo "    cli          - Starts an interactive CDAP CLI session"
-    echo "    tx-debugger  - Runs CDAP transaction debugger"
+    echo "    debug        - Runs CDAP debug functions"
```

Comment:
the command seems more generic (when there's only 1 debug function available - the tx one)